docker/Dockerfile

```diff
@@ -54,6 +54,7 @@ RUN useradd -ms /bin/bash celeryuser && usermod -a -G clamav celeryuser
 
 COPY app app
 COPY application.py .
+COPY entrypoint.sh .
```

Comment:
Could we rename this to `run_app_ecs.sh` and move it under `scripts/` to make it a bit more consistent with the location and the name of the other scripts?

The main reason is that `entrypoint.sh` by convention is used in the Dockerfile to define the `ENTRYPOINT` of the container, so only keeping it here would be a bit misleading - although I understand that we are using it as an entrypoint.